packages/core/rendering/core-rendering-server-internal/src/bootstrap/get_theme_tag.ts

```diff
@@ -7,18 +7,10 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-import type { ThemeVersion } from '@kbn/ui-shared-deps-npm';
-
 /**
  * Computes the themeTag that will be used on the client-side as `__kbnThemeTag__`
  * @see `packages/kbn-ui-shared-deps-src/theme.ts`
  */
-export const getThemeTag = ({
-  themeVersion,
-  darkMode,
-}: {
-  themeVersion: ThemeVersion;
-  darkMode: boolean;
-}) => {
-  return `${themeVersion}${darkMode ? 'dark' : 'light'}`;
+export const getThemeTag = ({ name, darkMode }: { name: string; darkMode: boolean }) => {
```

Comment:
previously themeVersion was typed as a const, is there any reason to relax this to a string? Seems like it'd be better to type it as `type ThemeName = 'v8' | 'amsterdam'`

Reply:
You're right, I wanted to address this in a follow-up PR that adds the actual new theme since updating the type globally here would need more file changes and more codeowners. Let me know if you're okay leaving it as is for the next day or so